### ISSUE TITLE
Do not render example value under schema section for request body data in object-model.jsx

### ISSUE THREAD

**Issue by jimcward** [swagger-api/swagger-ui#8470]
### Content & configuration

Swagger/OpenAPI definition:
 ```yaml
 # N/A
 ```

 Swagger-UI configuration options:
 ```js
 SwaggerUI({
   // nothing relevant
 })
 ```

### Is your feature request related to a problem?

In the [object-model.jsx](https://github.com/swagger-api/swagger-ui/blob/master/src/core/components/object-model.jsx) file there is a section that defines and renders "infoProperties":

```JS
let infoProperties = schema
   .filter( ( v, key) => ["maxProperties", "minProperties", "nullable", "example"].indexOf(key) !== -1 )
```
...
```JSX
{
   infoProperties.size ? infoProperties.entrySeq().map( ( [ key, v ] ) => <Property key={`${key}-${v}`} propKey={ key } propVal={ v } propClass={ propClass } />) : null
}
```
As this is an object model, the example is likely to be something more complex, like json, the example tab shows and nicely formats this, whether an example is provided in the schema or auto-generated by swagger-ui. 

However, only when a specific example is provided in the schema, swagger-ui then also chooses to render this against the schema tab at the end but in a completely unformulated, flat, hard to read way.

### Describe the solution you'd like
Put simply, it would be good if example was not considered part of the infoProperties collection when dealing with object models. The example tab should be sufficient as the value rendered in the schema tab just ends up being a harder to read, unformatted copy of that same data.

### Describe alternatives you've considered
Custom CSS rules could be put into place to try and hide just that section, but in our usages, using Swashbuckle across a large number of ASP.NET Core microservices where this is all centrally configured for consistency, it would mean ensuring each service manually had the CSS assets file included with it

**Comment by Jan-Wilamowski-Clario** [comment]
I also just stumbled upon this. Especially with larger examples, this looks terrible and makes the containing div flow over. There could be at least an option to hide the examples on the schema tab although I'm not sure why you would even display it there in the first place.

Any chance this could be added?
